deploy/cluster-manager/olm-catalog/cluster-manager/0.1.0/cluster-manager.v0.1.0.clusterserviceversion.yaml

```diff
@@ -232,9 +266,16 @@ spec:
     type: AllNamespaces
   keywords:
   - open-cluster-management
+  - ClusterManager
+  links:
+  - name: Cluster Manager Operator
+    url: https://github.com/open-cluster-management/registration-operator
   maintainers:
-  - {}
+    - {}
```

Comment:
There are validations that will demand at least one email address here.

Reply:
done